### PR TITLE
Group chat: stop watching of group's topic & handle new participant

### LINKED ISSUE #263: Stop watching group chat after leaving
Currently, user continues to watch group's messages, in result, sender continues to receive `ack` messages from the user that has left the chat.

### LINKED ISSUE #265: Adding of the new user to the chat is not handled
When the group admin adds a new user to the chat other members don't react to this update.

### PR DESCRIPTION
**Pull request by rasom**
fixes #265 
fixes #263

### CODE REVIEW

File: src/status_im/protocol/handlers.cljs [hunk 1]
```diff
@@ -246,7 +248,7 @@
                                    :delivery-status :pending}))))
   (fn [db [_ {:keys [type id to groupd-id]}]]
     (if (#{:message :group-message} type)
-      (let [chat-id (or groupd-id to)
+      (let [chat-id        (or groupd-id to)
```
Comment (alwx) [inline src/status_im/protocol/handlers.cljs]: typo?

Comment (alwx) [inline src/status_im/protocol/handlers.cljs]: typo?

File: src/status_im/protocol/listeners.cljs [hunk 2]
```diff
@@ -24,6 +25,11 @@
     (decrypt key content)
     content))
 
+(defn normalize-identity [identity]
+  (if (and identity (s/starts-with? identity "0x"))
+    (subs identity 2)
+    identity))
```
Comment (alwx) [inline src/status_im/protocol/listeners.cljs]: hm
there are also duplicates of this function in `status-im.models.pending-messages` (line 22) and `status-im.transactions.subs` (line 22). maybe we should use this function everywhere instead? (it is not related to this task, but anyway)

File: src/status_im/protocol/web3/delivery.cljs [hunk 3]
```diff
@@ -89,7 +89,7 @@
          (fn [messages]
            (when messages
              (let [message (messages id)
-                   message' (dissoc message to)]
+                   message' (dissoc message to nil)]
```
Comment (alwx) [inline src/status_im/protocol/web3/delivery.cljs]: why?

Comment (rasom) [inline src/status_im/protocol/web3/delivery.cljs]: This is full of magic at the moment: it is possible that the message with the same id will be sent to different users, that's why when `ack` comes we have to remove pending message by both `message-id` and `from`. But there are scenarios when the message is not sent to some particular user, like group message (at least it works so now), then the recipient is `nil`... Imagine, we send group message and it is added to pending messages map with `[message-id nil]` key, so when `ack` is received we are trying to remove `[message-id from]` and in result the message is not removed from pending messages collection.

Comment (alwx) [inline src/status_im/protocol/web3/delivery.cljs]: ok, got it

**Pull request merged**